website/docs/migrating-from-docker/customizing-docker-compatibility.md

```diff
@@ -0,0 +1,177 @@
+
+Podman Desktop displays the information of the Podman machine that emulates the Docker socket on the Docker compatibility page.
+
+**_On Windows_**
+
+The binding between the Podman machine and the system socket is not known. So, Podman Desktop displays only the information of the Docker socket on the Docker compatibility page.
+
+## Enable Docker compatibility
+
+#### Prerequisites
+
+- [A running Podman machine](/docs/podman/creating-a-podman-machine).
+
+<Tabs>
+   <TabItem value="macOS" label="macOS" className="markdown">
+
+#### Procedure
+
+1. Click **Docker Compatibility** in the status bar. A notification opens.
+1. Click **Enable**.
+   ![enable docker compatibility](img/enable-docker-compatibility.png)
+1. Enter your machine password when prompted. A successful operation notification opens.
+1. Click **OK**. A notification prompts you to restart your Podman machine.
+1. Click **Yes**.
+   ![restart Podman machine](img/restart-podman-machine.png)
+1. Perform one of the following steps: - Go to **Settings > Experimental**, and click the toggle button.
+   ![enable toggle button - Experimental](img/enable-using-experimental.png) - Go to **Settings > Preferences > Experimental (Docker Compatibility)**, and click the toggle button.
+   ![enable toggle button - Preferences](img/enable-using-preferences.png)
+   A Docker Compatibilty section is added to the list of **Settings**.
+
+#### Verification
+
+Perform any of the following steps:
+
+- Check whether the Docker socket is a symbolic link for the Podman socket:
+
+  ```shell-session
+  $ ls -la /var/run/docker.sock
+  ```
+
+  The output points to a `podman.sock` file, as shown below:
+
+  ```shell-session
+  /var/run/docker.sock -> /Users/username/.local/share/containers/podman/machine/podman.sock
+  ```
+
+- Run the following command to check the output returns the Podman version rather than the Docker version:
+
+  ```shell-session
+  $ curl -s --unix-socket /var/run/docker.sock "http://v1.41/info"  | jq -r .ServerVersion
+  ```
+
+- Run the `docker context list` command to check that the Docker CLI context is set to `unix:///var/run/docker.sock`.
+
+</TabItem>
+<TabItem value="win" label="Windows/Linux" className="markdown">
+
+#### Procedure
+
+- Perform one of the following steps:
+
+  - Go to **Settings > Experimental**, and click the toggle button.
+  - Go to **Settings > Preferences > Experimental (Docker Compatibility)**, and click the toggle button.
+
+  A Docker Compatibilty section is added to the list of **Settings**.
+
+#### Verification
+
+Perform any of the following steps:
+
+- Run the following command to check the output returns the Podman version rather than the Docker version:
+
+  ```shell-session
+  $ docker info --format=json | jq -r .ServerVersion
+  ```
+
+- Run the `docker context list` command to check that the Docker CLI context is set to the default value:
+  - Linux: `unix:///var/run/docker.sock`
```

Comment:
I don't like how we combine windows and linux sections.

Could we separate these? It feels like you have to read a lot of info just to find about this Linux section.